app/components/Combobox.tsx

```diff
@@ -38,6 +38,7 @@ interface ComboboxProps<T> {
   clearsInputOnFocus?: boolean;
   onChange?: (selectedOption?: ComboboxOption<T>) => void;
   fullWidth?: boolean;
+  fuseOptions?: object;
```

Comment:
There is pretty much never a good reason to use the `object` type in TS. When I hover over the "new Fuse" i can see what types it actually allows:

![image](https://user-images.githubusercontent.com/38327916/198870110-75291ce7-e044-4827-bec0-a642910ac52b.png)

So this could be:

```suggestion
  fuseOptions?: Fuse.IFuseOptions<ComboboxOption<T>>;
```

Reply:
I should definitely get used to highlighting to extract data types more in VSCode - thank you Sendou!